third_party/boringssl/LICENSE

```diff
@@ -0,0 +1,221 @@
+BoringSSL is a fork of OpenSSL. As such, large parts of it fall under OpenSSL
```

Comment:
This is not the right license for this directory.... The files in this directory are just the METADATA and owners files.

We should instead update the license tool to ignore this directory.